src/rolodex/rolodexPrinter.js

```diff
@@ -1,11 +1,10 @@
 import people from './people.json';
 
-people.forEach(function (person) {
-  const names = person.name.split(' ');
-  const firstName = names[0];
-  const lastName = names[1];
-  const email = person.email;
-  const phone = person.phone;
-
-  console.log('First name: ' + firstName + '\nLast name: ' + lastName + '\nEmail: ' + email + '\nPhone number: ' + phone + '\n');
+people.forEach((person) => {
+  const { name } = person;
+  const { firstName, lastName } = name.split(' ');
```

Comment:
`firstName` and `lastName` will be `undefined` here because object destructuring is used instead of array destructuring. That can be fixed by doing:

```js
const [ firstName, lastName ] = name.split(' ');
```